hackforla/VRMS
# Turn off int tests

HOLD AND DO NOT MERGE

Overview:
The integration tests that I wrote were functional when I implemented them, but they do not work with the cron job worker files that were reimplemented after I added my tests. The cron based processes are breaking all of the tests and mocking out all of these files is not a winning strategy for test maintainability. After reviewing these changes, my suggestion is to either remove these tests or to refactor completely the cron based tasks.

How to verify:
1. Run the tests in the backend. Note that the processes hang. 
2. Go to the backend/app.js file and comment out the cron process files WORKERS section. 
3. Run the tests and note that they function. 

Please note:
This PR was based on another branch. https://github.com/hackforla/VRMS/pull/339. If you merge this first branch, then the changes will be very limited on this one.
